go.mod

```diff
@@ -279,8 +281,8 @@ require (
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 
-	// Our wasmd branch is: https://github.com/desmos-labs/wasmd v0.28.x-desmos
-	github.com/CosmWasm/wasmd => github.com/desmos-labs/wasmd v0.29.0-desmos.2
+	// Our wasmd branch is: https://github.com/desmos-labs/wasmd v0.30.x-desmos
+	github.com/CosmWasm/wasmd => github.com/desmos-labs/wasmd v0.23.1-0.20230202060518-edbdd2dcb064
```

Comment:
It's probably better to tag a version here before merging this PR. So we can use `v0.30.0-desmos.1` instead of the commit hash